Notable changes (#2429)
kyma-project/kyma-environment-broker: docs/assets/notable-change-template.md

# Updating Kyma Environment Broker

> > [!NOTE]

Reviewer: ```suggestion
> [!NOTE] <!-- In case of a mandatory update that will cause unwanted/severe impact if not performed, use !WARNING instead of !NOTE.-->
```

> > Describe whether the notable change is mandatory or optional.

Reviewer: ```suggestion
> <!--State whether the notable change is mandatory or optional. Briefly explain what will happen without the update or why it's recommended.-->
```

## Prerequisites

> Describe the generic prerequisites common for all the components of the Kyma Control Plane.

Reviewer: ```suggestion
Describe the generic prerequisites common for all the Kyma Control Plane (KCP) components.
```
